### PR TITLE
MGDOBR-47: add demo and filters documentation

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-47](https://issues.redhat.com/browse/MGDOBR-47)

This PR aims to add the instructions for a demo.  

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: DEMO.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+The application is now `AVAILABLE` and we also have the information about the endpoint to use to push the events: `/ingress/events/87508471-ee0f-4f53-b574-da8a61285986` in this particular case.
+
+## How to add Processors to the Bridge
+
+It is possible to add Processors to the Bridge instance using the endpoint `/api/v1/bridges/{bridgeId}/processors`. 
+There are 3 basic concepts around a Processor:
+
+- **filters**: when one or more filters are specified, only those events that evaluates the filters to `true` will be processed by the specified action. The documentation around the filters can be found [here](FILTERS.md).
+- **transformations**: if a transformation schema has been specified in the request, the original event is transformed according to the schema before it is sent to the action.
+- **actions**: the action to take with the filtered events.
```
Comment (danielezonca) [inline DEMO.md:74]: I would prefer to switch the order: transformation concept is tied to action. Wdyt?
```suggestion
- **actions**: the action to take with the filtered events.
- **transformations**: an optional transformation logic can be specified for each action definition.
```

File: DEMO.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
+# DEMO
```
Comment (danielezonca) [inline DEMO.md:1]: Ok it is Friday afternoon and it is also beer o'clock but I was thinking to convert this file (and filters too) as a jupyther notebook and execute it as part of CI job during PR checks to making sure it is still working 😃 
It is mainly a set of curl/REST calls to be converted as Python code :)
Wdyt? 
image

Comment (r00ta) [reply DEMO.md:1]: Makes sense, I've created https://issues.redhat.com/browse/MGDOBR-68

**Review by danielezonca — APPROVED** [review]
LGTM 👍

**Review by robpblake — APPROVED** [review]
Thanks for writing this up 🍻

**Pull request merged**